src/images/XSVG.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+
+import colors from '../styles/colors';
+
+const Svg = styled.h4`
+	color: ${colors.expired700};
+`;
+
+const XSVG = () => <Svg>Expired</Svg>;
```

Comment:
Much too scary and agressive.  Let's set the default colour to something a little less `danger` looking, and set the message to something a little clearer, is `Invoice expired`

Reply:
e14befe